### ISSUE TITLE
Add way to manually verify user account

### ISSUE THREAD

**Issue by pippinsplugins** [easydigitaldownloads/Easy-Digital-Downloads#3877]
We should add a row action to the Users page (and/or the customer screen) to allow site admins to manually verify an account that is flagged as pending verification.

**Comment by cklosowski** [comment]
@pippinsplugins thoughts? Just an action on the user list table?
![screen shot 2015-10-06 at 4 45 24 pm](https://cloud.githubusercontent.com/assets/1390124/10325819/aa3e98f0-6c49-11e5-800e-4e0e0d738b34.png)

**Comment by cklosowski** [comment]
That should probably say `Verify` instead of `Activate`

**Comment by chriscct7** [comment]
I was going to say the same thing

**Comment by pippinsplugins** [comment]
Looks great and I agree on `Verify` over `Activate`.

**Comment by mindctrl** [comment]
I like `Verify`, but on the users list, both terms are a bit ambiguous. Verify and Activate could be mistaken for the functions of another plugin, like something a membership or user registration plugin would do.

What we're really doing is verifying a user's access to past payments. The user account is technically already verified by WordPress. At first I thought it made more sense in the Customers list, but that doesn't feel right either.

What do you think about not having it on the action row and moving it into the user account screen somewhere? We can be more descriptive about the action about to be taken since `Verify` doesn't provide any context. Hopefully this is something admins won't be doing often.

**Comment by pippinsplugins** [comment]
@mindctrl you raise some good points.

What about both on the user screen an the customer screen.

One of the concerns I have with all of the above options is that none of them make it clear to the admin that an account is pending verification for access to purchase history. If we put it on customer or user details, we can make it big and bold so ti's very apparent.

**Comment by cklosowski** [comment]
Sounds good, On the user and customer screen it is.

**Comment by cklosowski** [comment]
Actually, I disagree with it being a 'user' setting. Since it's hard to make it sure that it's part of EDD, I think it should only be on the customer, not the user at all. It's too ambiguous that we're 'verifying the user' when WordPress would have already done that.

**Comment by cklosowski** [comment]
![screen shot 2015-10-07 at 6 12 46 am](https://cloud.githubusercontent.com/assets/1390124/10338927/7efd2c14-6cbd-11e5-86a3-b86ac6950aff.png)
![screen shot 2015-10-07 at 6 34 20 am](https://cloud.githubusercontent.com/assets/1390124/10338926/7ef98780-6cbd-11e5-875f-113bcbce1a19.png)

**Comment by cklosowski** [comment]
@mindctrl @pippinsplugins @chriscct7 thoughts on that? All from the customers interface, list table and customer view itself.

**Comment by cklosowski** [comment]
I want to add a `What's this mean?` link into the notice, but couldn't find anything in the docs to explain this...maybe we should get a doc on it?

**Comment by mindctrl** [comment]
That looks great. Almost as great as the dude in the mirror.

**Comment by pippinsplugins** [comment]
A couple of thoughts:

1. I like the notice
2. I agree that we need a "what's this?". We could get a doc written up quickly.
3. I agree (now) on not showing it on the user screen

**Comment by cklosowski** [comment]
@topher1kenobe can we get a doc written up for the user verification process? We want to link to it in the admin notice with a `What's this?` link.

**Comment by spencerfinnell** [comment]
I think the action in the notice should be `Verify account` and not `click here` -- like an action on other notices: http://www.screensnapr.com/0jzo3hrwbwd8

**Comment by cklosowski** [comment]
@spencerfinnell Yeah working out the details on the messaging, we're going to write up a doc to explain the process as well.

**Comment by cklosowski** [comment]
@mindctrl would you mind testing this out a bit? We're still waiting on the docs, but the functionality shouldn't change.

**Comment by mindctrl** [comment]
@cklosowski did you PR this? I don't see a branch for it.

**Comment by cklosowski** [comment]
@mindctrl helps If I actually commit stuff, instead of just stash it. Enjoy :)

**Comment by pippinsplugins** [comment]
@cklosowski Doc to link to: http://docs.easydigitaldownloads.com/article/1031-user-verification-for-easy-digital-downloads

**Comment by mindctrl** [comment]
@cklosowski this is working great here.

**Comment by pippinsplugins** [comment]
This is working perfectly for me. I made a couple of small changes (mostly code location). Ready for final review and merge @cklosowski

**Comment by wpinventory** [comment]
Thank you for all you guys do over there to power our website.  We love everything you guys do and all the documentation you supply for your product base.  I can just about always find what I am looking for and if not; your support team is always amazing!  I know this thread is old now, but I just found it super useful in helping a customer that just could not get an email from my website.  Not sure how he gets the download emails.... but, this helped me with this particular customer.  Well done team!
